consensus/replay_test.go

```diff
@@ -1104,6 +1125,17 @@ func (bs *mockBlockStore) LoadSeenCommit(height int64) *types.Commit {
 	return bs.commits[height-1]
 }
 
+func (bs *mockBlockStore) PruneBlocks(height int64) (uint64, error) {
+	pruned := uint64(0)
+	for i := int64(0); i < height-1; i++ {
+		bs.chain[i] = nil
+		bs.commits[i] = nil
+		pruned++
```

Comment:
aren't pruned always equals to `height-1`?

Reply:
I suppose. Normally it will be newBase - oldBase, but the mock generally always starts from the first.